rust-bindings/sys/Cargo.toml

```diff
@@ -71,7 +72,7 @@ license = "MIT"
 links = "ostree-1"
 name = "ostree-sys"
 repository = "https://github.com/ostreedev/ostree-rs"
-version = "0.9.2"
+version = "0.10.0"
```

Comment:
Side note: now that this is in-tree, it seems like we should release them together and have the version string reflect the libostree release. To make it semver, we can add `0.` in front for now, and eventually `1.`, etc...